onload/kmm/Dockerfile

```diff
@@ -1,8 +1,10 @@
 # SPDX-License-Identifier: MIT
 # SPDX-FileCopyrightText: (c) Copyright 2023 Advanced Micro Devices, Inc.
 ARG DTK_AUTO
+ARG DTK_IMAGE
+ARG UBI8_IMAGE
 
-FROM ${DTK_AUTO} as builder
+FROM ${DTK_IMAGE:-${DTK_AUTO}} as builder
```

Comment:
AFAIR, this `FROM` statement might make docker "forget" `ARG UBI8_IMAGE` used in the next `FROM`. Why is it not the case here? Also, noticed the same in diagnostics.

Reply:
My understanding (which could be wrong) is that the `ARG` parameters which happen before the first `FROM` act more like global variables.
https://docs.docker.com/engine/reference/builder/#understand-how-arg-and-from-interact

Reply:
Thanks, this part of the docs seems to be written specifically for such occasions!